python/py-apache-beam/Portfile

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-apache-beam
+version             2.16.0
+platforms           darwin
+license             Apache-2
+maintainers         nomaintainer
+
+description         Apache Beam SDK for Python
+
+long_description    Apache Beam is a unified model for defining both\
+                    batch and streaming data-parallel processing\
+                    pipelines, as well as a set of language-specific\
+                    SDKs for constructing pipelines and Runners for\
+                    executing them on distributed processing backends,\
+                    including Apache Apex, Apache Flink, Apache Spark,\
+                    Google Cloud Dataflow and Hazelcast Jet.
+
+homepage            https://beam.apache.org
+master_sites        pypi:a/apache-beam
+use_zip             yes
+extract.mkdir       no
+distname            apache-beam-${version}
+
+checksums           rmd160  dc243db6c9fffab8cc82f1d62822fb3eae5b983b \
+                    sha256  1843d0cfc9f945e90d21e3d94b59e82613d0d5cbc44ae7befa65872f114e6b94 \
+                    size    1621014
+
+python.versions     27 35 36 37 38
```

Comment:
Let's only go with the latest python versions for new ports. So remove py27, 35, and 36.

Reply:
Thanks done.